pkg/translator/prometheus/README.md

```diff
@@ -22,7 +22,7 @@ List of transformations performed on OpenTelemetry metrics names:
 | Case                                                     | Transformation                                                                                                                   | Example                                                                                     |
 | -------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------- |
 | Unsupported characters and extraneous underscores        | Replace unsupported characters with underscores (`_`). Drop redundant, leading and trailing underscores.                          | `(lambda).function.executions(#)` ==> `lambda_function_executions`                          |
-| Standard unit                                            | Convert the unit from [Unified Code for Units of Measure](http://unitsofmeasure.org/ucum.html) to Prometheus standard and append | `system.filesystem.usage` with unit `By` ==> `system_filesystem_usage_bytes`                |
+| Standard unit                                            | Convert the unit from [Unified Code for Units of Measure](http://unitsofmeasure.org/ucum.html) to Prometheus standard and append | `system.filesystem.usage` with unit `bytes` ==> `system_filesystem_usage_bytes`                |
```

Comment:
I don't think this is a type. `By` unit is set in [the host receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/metadata.yaml#L31) for this metric according to the OTel spec